apps/voting-disputable/test/helpers/voting.js

```diff
@@ -1,14 +1,43 @@
 const { bigExp } = require('@aragon/apps-agreement/test/helpers/lib/numbers')
+const { encodeCallScript } = require('@aragon/contract-test-helpers/evmScript')
+const { decodeEventsOfType } = require('@aragon/apps-agreement/test/helpers/lib/decodeEvent')
```

Comment:
This is honestly getting ridiculous—these should all be coming from the same place (https://github.com/aragon/contract-helpers/issues/27).

Reply:
I know, I'm working on a first step to start getting these fixed

Reply:
Opened https://github.com/aragon/contract-helpers/pull/35 and https://github.com/aragon/aragon-apps/pull/1198 on top of it to remove all the duplicated logic from Agreements, I promise to update disputable voting app when these two are merged.